### PR TITLE
Do you speak PrestaShop? January 17 Edition

### PR DESCRIPTION
**Pull request by AlexEven**

### CODE REVIEW

File: news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+icon: icon-world
+tags: [translation, i18n, l10n]
+---
+
+You've probably been anxiously wondering what happened with the [PrestaShop translation project](https://crowdin.com/project/prestashop-official) this month? Well, here is the first monthly report of 2017, just for you! :)
+
+## Project news
+
+
+### 1.7.1 is on its way
```
Comment (xBorderie) [inline news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown:16]: 1.7.1.0 ?

File: news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+We're getting close to the upload of new strings for 1.7.1 version. We'll publish a short article here when it's done, to let you know what's new!
```
Comment (xBorderie) [inline news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown]: "close to uploading new strings"

File: news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+
+### Translations update
+
+As usual, 1.7 translations are updated each Monday, and 1.6 translations every two weeks (or so ^^).
```
Comment (xBorderie) [inline news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown]: each / every. Pourquoi pas "every" partout ?

File: news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+* **Arabic** (55% translated vs 28% validated in 1.6)
+* **Galician** (74% vs 52% in 1.6)
+* **Catalan** (100% translated vs 82% validated in 1.7)
+* **Portuguese**, Brazil (100% vs 85% in 1.7)
+* **Finnish** (100% vs 85% in 1.6)
+
+![Languages that need proofreading](/assets/images/2017/02/Build_Crowdin_proofreading_Jan17.png)
+
+Again, these figures are for 1.7, but 1.6 also need some proofreading too :)
```
Comment (xBorderie) [inline news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown]: "also needs"?

Comment (xBorderie) [reply news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown]: also/too: redondance ?

File: news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+If you wish to help to proofread what has been translated, please [contact me on Crowdin](https://crowdin.com/profile/alex-even) with the language you’d like to proofread. We need your help!
+
+If you haven’t joined us on Crowdin yet, [it's never too late](https://crowdin.com/project/prestashop-official)!
```
Comment (xBorderie) [inline news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown]: ’ => ' ? :p

File: news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown (hunk 3 above)
Comment (AlexEven) [reply news/_posts/translation/2017-02-10-do-you-speak-prestashop-jan-2017.markdown]: Parce que j'aime bien varier :p

**Pull request merged**